src/style.css

```diff
@@ -1 +1,99 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: normal;
+}
+
+.body {
+  font-family: Roboto, serif;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  justify-content: start;
+  align-items: flex-start;
+  height: 100vh;
+  width: 100%;
+}
+
+.search-panel {
+  position: relative;
+  width: 100%;
+  max-width: 900px;
+  padding: 20px;
+}
+
+.search__input {
+  width: 100%;
+  font-size: 16px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  transition: all 0.3s ease;
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.search__input--big {
+  background-image: url(images/Search.svg);
+  background-position: 26px 25px;
+  background-repeat: no-repeat;
+  padding-top: 25px;
+  padding-left: 62px;
+  padding-bottom: 26px;
+  width: 100%;
+  height: 70px;
+  opacity: 1;
+}
+
+.search__input--small {
+  background-image: url(images/Search.svg);
```

Comment:
The `background-image` URL should be a relative path to the image within the project directory. Ensure that the path is correct and accessible from the CSS file.